src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
It would be nice to have some tests

Reply:
It would have been nice, but time isn't on my side. I'm also just learning React and am happy to have completed what was required of me.